src/main/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/BulkPrintTransferJob.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.cmc.claimstore.jobs.cron;
+
+import lombok.Getter;
+import org.quartz.DisallowConcurrentExecution;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.services.BulkPrintTransferService;
+import uk.gov.hmcts.cmc.scheduler.model.CronJob;
+
+@Component
+@Getter
+@DisallowConcurrentExecution
+public class BulkPrintTransferJob implements CronJob {
+
+    @Value("${claimsReadyForTransfer.schedule:0 * * ? * *}")
```

Comment:
I dont think we should add a default here. We should explicitly configure the cron expression so we can control if the job gets run or not.

Reply:
We will need to add the configuration to the charts. See the configuration for SCHEDULE_STATE_TRANSITION_STAY_CLAIM in values.yaml (and possibly other files too)

Reply:
I just added that for testing on local. Will remove that